pod/perlref.pod

```diff
@@ -117,6 +117,16 @@ brackets:
         'Clyde' => 'Bonnie',
     };
 
+But be aware:
+
+    $folks{A} = {
+        'Adam'  => 'Eve',
+        'Clyde' => 'Bonnie',
+    };
+    $folks{B} = $folks{A};           # just pointers to the anonymous hash
+    %{ $folks{C} } = %{ $folks{A} }; # deep copy
+    $folks{A}{Clyde} = 'Wilma';      # also updates B! C is immune
```

Comment:
"$folks{B}" isn't a _thing_, it's the name of a _reference_ to a thing.

It might be worth adding a whole paragraph up top about what it means for two references to point to the same structure (even if the reference is stored in different places) instead of including a 6 line code example in the middle of a sentence.